__tests__/app.test.js

```diff
@@ -74,6 +74,7 @@ describe('app.js', () => {
                         created_at: '2020-07-09T20:11:00.000Z',  //converting the unicode time was 1 hour wrong (daylight savings time?)
                         votes: 100,
                         article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                        comment_count: '11',
```

Comment:
When adding a new property to an endpoint response, we should write a new test. We should never go back to edit old tests 👍